packages/devops/scripts/deployment/setupGoldMaster.sh

```diff
@@ -51,7 +44,15 @@ mkdir -p /home/ubuntu/.local/share/lpass
 
 # clone our repo
 cd /home/ubuntu
-sudo -Hu ubuntu git clone https://github.com/beyondessential/tupaia.git
+# TODO: Remove this once RN-508 is released
+sudo -Hu ubuntu git clone -b rn-508-upgrade-node-to-14 https://github.com/beyondessential/tupaia.git
+
+# install node and yarn
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+nvm install $(sudo cat tupaia/.nvmrc)
```

Comment:
This is a little change to ensure that our ec2 images use the same version of node as we specify in the `.nvmrc` (ie. the one we recommend devs use). So in the future we can just bump it there 👍